modules/storages/spec/features/storages/admin/project_storages_spec.rb

```diff
@@ -37,11 +37,30 @@
                with_flag: { enable_storage_for_multiple_projects: true } do
   shared_let(:admin) { create(:admin, preferences: { time_zone: "Etc/UTC" }) }
   shared_let(:non_admin) { create(:user) }
+
   shared_let(:project) { create(:project, name: "My active Project") }
-  shared_let(:archived_project) { create(:project, active: false, name: "My archived Project") }
-  shared_let(:storage) { create(:nextcloud_storage, name: "My Nextcloud Storage") }
-  shared_let(:project_storage) { create :project_storage, project:, storage: }
-  shared_let(:archived_project_project_storage) { create :project_storage, project: archived_project, storage: }
+  shared_let(:archived_project) do
+    create(:project,
+           active: false,
+           name: "My archived Project")
+  end
+  shared_let(:storage) do
+    create(:nextcloud_storage,
+           :as_automatically_managed,
+           name: "My Nextcloud Storage")
+  end
+  shared_let(:project_storage) do
+    create(:project_storage,
+           project:,
+           storage:,
+           project_folder_mode: "automatic")
+  end
+  shared_let(:archived_project_project_storage) do
+    create(:project_storage,
+           project: archived_project,
+           storage:,
+           project_folder_mode: "inactive")
+  end
```

Comment:
🟢  _nit_: I personally prefer the compact one-liner approach where possible, reduces "setup fatigue" IMO. The faster we can get to the tests without scrolling, the better 😄

Reply:
I somewhat agree - it depends on how long the line gets. Sometimes it's also easier to see what the differences are between certain test objects. But if you only need 2 or 3 short arguments, I'd also go for oneliners :+1: